hyrisecockpit/api/app/workload/service.py

```
@@ -52,69 +44,81 @@ def create(cls, interface: WorkloadInterface) -> Optional[Workload]:
         Returns the created Workload.
         Returns None if a Workload with the given ID already exists.
         """
-        response = cls._send_message(
-            Request(header=Header(message="start workload"), body=dict(interface))
-        )
-        return (
-            None
-            if response["header"]["status"] == 409
-            else Workload(**response["body"]["workload"])
+        response = cls._send_message_to_dbm(
+            Request(header=Header(message="start worker"), body={})
         )
+        if response["header"]["status"] == 200:
+            response = cls._send_message_to_gen(
+                Request(header=Header(message="start workload"), body=dict(interface)),
+            )
+            return (
+                None
+                if response["header"]["status"] == 409
+                else Workload(**response["body"]["workload"])
+            )
+        else:
+            return None
```

Like mentioned before we could return the error code as an integer directly to the controller 🙃.